custom/gcc_sangath/rules/custom_actions.py

```diff
@@ -58,15 +58,15 @@ def _get_case_updates(peer_rating_cases):
 
     latest_peer_review = _get_latest_peer_review_date(peer_rating_cases)
 
-    case_updates = {
+    case_updates = dict({
```

Comment:
What difference does this change make?

Reply:
I am not sure. IDE complaint about it. I am guessing that is a best practice.